SentryTestUtilsTests/TestContstantTests.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+@testable import SentryTestUtils
+import XCTest
+
+class TestContstantTests: XCTestCase {
+    func testDsnForTestCase_localTargetClass_shouldUseTypeNameInDSN() {
+        // -- Arrange --
+        class FooTests: XCTestCase {}
+
+        // -- Act --
+        let dsn = TestConstants.dsnForTestCase(type: FooTests.self)
+
+        // -- Assert --
+        XCTAssertEqual(dsn, "https://FooTests:password@app.getsentry.com/12345")
+    }
+
+    func testDsnForTestCase_externalFrameworkObjectiveCClass_shouldUseTypeNameInDSN() {
+        // -- Arrange --
+        class FooTests: XCTestCase {}
```

Comment:
`l`: FooTests isn't used here. Same applies for testDsnForTestCase_externalFrameworkSwiftClass_shouldUseTypeNameInDSN
```suggestion
```